scss/components/modal.scss

```diff
@@ -70,13 +77,20 @@ $block: #{$fd-namespace}-modal;
     height: fd-space(9);
     @include fd-focus();
   }
+
   &__body {
     max-height: calc(100vh - #{$fd-modal-header-height} - #{$fd-modal-footer-height});
     overflow-y: auto;
-    > p {
-      padding: 0;
+
+    > :first-child {
```

Comment:
Added these just for top and bottom margin; everything else should be styled at the (incoming content) child level.

Reply:
Do we need something like: 
```
> * {
      @include fd-reset();
    }
``` 

here as well so the un-normalize sheet can't affect the interior of the `fd-modal__body`? Or should we allow that?

Reply:
That _seems_ like a bit much (at least at this time) until we know more about how self-contained styles will affect overall page and application styling.